monitoring/airgap/cluster/airgap-values-prom-operator.yaml

```diff
@@ -0,0 +1,47 @@
+global:
+  imageRegistry: __AIRGAP_REGISTRY__
+  imagePullSecrets:
+    - __AIRGAP_IMAGE_PULL_SECRET_NAME__
+
+## Individually update image registries
+# prometheusOperator:
+#  image:
+#    registry: __AIRGAP_REGISTRY__
+#  prometheusConfigReloader:
+#    image:
+#      registry: __AIRGAP_REGISTRY__
+
+# kube-state-metrics:
+#  image:
+#    registry: __AIRGAP_REGISTRY__
+
+# =======================
+# Prometheus AlertManager
+# =======================
+# alertmanager:
+#  alertmanagerSpec:
+#    image:
+#      registry: __AIRGAP_REGISTRY__
```

Comment:
You're right.  This was more of a testing effort when I was running into an issue with the global setting.  Since I eventually got the global parameters working, we can delete the individual ones.  I'll fix that in my next push.